conan/clang-tools-extra/conanfile.py

```diff
@@ -48,16 +43,15 @@ class ClangToolsExtraConan(ConanFile):
 
     def source(self):
         git_cache = os.environ.get("CONAN_LLVM_GIT_CACHE")
-        cache_hit = lambda: os.path.exists(f"{git_cache}/.git")
+        cache_hit = os.path.exists(f"{git_cache}/.git")
         cache_arg = f" --reference-if-able '{git_cache}' " if git_cache else ""
 
         if git_cache and cache_hit():
```

Comment:
You converted `cache_hit` from a function to a boolean. So, I think:
```suggestion
        if git_cache and cache_hit:
```

Reply:
Fixed in dc60201d2a76.